.github/workflows/sync.yml

```
@@ -0,0 +1,44 @@
+          # Get all remote branch names (exclude HEAD/refs)
+          branches=$(git ls-remote --heads upstream | awk '{print $2}' | sed 's|refs/heads/||')
+          # Store as newline-separated text (skip JSON conversion)
+          echo "branches<<EOF" >> $GITHUB_OUTPUT
+          echo "$branches" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
+      - name: Sync Branches
+        run: |
+          echo "Syncing branches:"
+          echo "${{ steps.get-branches.outputs.branches }}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)